src/compiler/types.ts

```diff
@@ -8235,12 +8235,13 @@ export const enum OuterExpressionKinds {
     Parentheses = 1 << 0,
     TypeAssertions = 1 << 1,
     NonNullAssertions = 1 << 2,
-    PartiallyEmittedExpressions = 1 << 3,
+    SatisfiesExpressions = 1 << 3,
+    PartiallyEmittedExpressions = 1 << 4,
 
     Assertions = TypeAssertions | NonNullAssertions,
-    All = Parentheses | Assertions | PartiallyEmittedExpressions,
+    All = Parentheses | Assertions | SatisfiesExpressions | PartiallyEmittedExpressions,
 
-    ExcludeJSDocTypeAssertion = 1 << 4,
+    ExcludeJSDocTypeAssertion = 1 << 5,

```

Comment:
That we keep changing this makes me wonder if we should just move this (and `PartiallyEmittedExpressions`) up to the end of the bits or something.

Reply:
I think I was suggesting like:

```ts
export const enum OuterExpressionKinds {
    Parentheses = 1 << 0,
    TypeAssertions = 1 << 1,
    NonNullAssertions = 1 << 2,
    SatisfiesExpressions = 1 << 3,
    PartiallyEmittedExpressions = 1 << 4,

    Assertions = TypeAssertions | NonNullAssertions,
    All = Parentheses | Assertions | SatisfiesExpressions | PartiallyEmittedExpressions,
    ExcludeJSDocTypeAssertion = 1 << 29,
}
```

But, this isn't really a request for this to be changed. Just thinking aloud.